tests/js/karma.conf.js

```diff
@@ -91,6 +91,6 @@ module.exports = function(config) {
 
     // Continuous Integration mode
     // if true, Karma captures browsers, runs the tests and exits
-    singleRun: true
+    singleRun: false
```

Comment:
Why this change?  Shouldn't this be true when running the Karma tests on Travis?